seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -5411,7 +5411,7 @@
         "Now, take a look at a <code>do...while</code> loop.",
         "<blockquote>var ourArray = []; <br>var i = 5;<br>do {<br>  ourArray.push(i);<br>  i++;<br>} while (i < 5);</blockquote>",
         "In this case, we initialize the value of <code>i</code> as 5, just like we did with the while loop. When we get to the next line, there is no check for the value of <code>i</code>, so we go to the code inside the curly braces and execute it. We will add one element to the array and increment <code>i</code> before we get to the condition check. Then, when we get to checking if <code>i < 5</code> see that <code>i</code> is now 6, which fails the conditional check. So we exit the loop and are done. At the end of the above example, the value of <code>ourArray</code> is <code>[5]</code>.",
-        "Essenially, a <code>do...while</code> loop ensures that the code inside the loop will run at least once.",
+        "Essential, a <code>do...while</code> loop ensures that the code inside the loop will run at least once.",
```

Comment:
Essential => Essentially

Reply:
Thanks for the notice, i let myself be guided by the correction in the issue but i already corrected it to "Essentially".